SparkPost/2web2ui
# FE-888 Hibana Snackbar and Utility Function

### What Changed
 - Added a utility function to remove styled system props before displaying OGMatchbox components
 - Added a wrapper for the hibana Snackbar component
 - Replaced the usages of Snackbar with the wrapper in:
   - ConsentBar
   - Alert
 - Updated the ProgressBar wrapper to use the utility function and updated tests

### How To Test
 - Ensure Alerts look good (can trigger those in various places throughout the app, success, danger, etc.)
 - Test that the consent bar looks right - The hibana view will never be seen in production, technically because you can't login without consenting and the default is that hibana is off, but you can test it manually by editing the `HibanaContext.isHibanaEnabled` default to be true locally
